src/libint_3center.F

```diff
@@ -94,13 +109,26 @@ SUBROUTINE eri_3center(int_abc, la_min, la_max, npgfa, zeta, ra, &
                                                             b_offset, b_start, c_offset, c_start, &
                                                             i, ipgf, j, jpgf, k, kpgf, li, lj, lk, &
                                                             ncoa, ncob, ncoc, p1, p2, p3
-      REAL(dp)                                           :: my_omega, my_r_cutoff, zeti, zetj, zetk
+      REAL(dp)                                           :: dr_ab, dr_ac, dr_bc, my_omega, &
+                                                            my_r_cutoff, zeti, zetj, zetk
       REAL(dp), DIMENSION(:), POINTER                    :: p_work
       TYPE(params_3c), POINTER                           :: params
 
       NULLIFY (params, p_work)
       ALLOCATE (params)
 
+      dr_ab = 0.0_dp
+      dr_bc = 0.0_dp
+      dr_ac = 0.0_dp
+
+      IF (op == do_potential_truncated .OR. op == do_potential_short) THEN
+         dr_bc = r_cutoff*1.01
+         dr_ac = r_cutoff*1.01
```

Comment:
Maybe a comment as to why a larger effective cutoff is required could be useful for the future ?

Reply:
This is needed due to the discontinuity of the truncated Coulomb potential at the truncation radius, this factor is now a properly documented constant in `libint_2c_3c` (you may want to use this too for the neighbor lists in `xas_tdp_utils`).